content/kotlin/concepts/datetime/terms/todayIn/todayIn.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: '.todayIn()' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'Passing a specific time zone as an argument and calling this method on an instant value of Clock will return the current date without time.' # Required; ideally under 150 characters and starts with a present-tense verb (used in search engine results and content previews)
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Methods'
+  - 'Date'
+  - 'Kotlin'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **`.todayIn()`** method takes a `TimeZone` argument, and when called on an instant value of `Clock`, it returns a local date without time.
+
+## Syntax
+
+
+```
+Clock.todayIn(timeZone: TimeZone)
+```
```

Comment:
Please go through the [style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md) and format the code blocks and syntax as mentioned.

Reply:
I reviewed the style guide and made the necessary corrections to the todayIn.md file.